FirebaseRemoteConfig/Sources/Private/RCNConfigFetch.h

```diff
@@ -30,6 +30,11 @@ NS_ASSUME_NONNULL_BEGIN
 /// Completion handler invoked by NSSessionFetcher.
 typedef void (^RCNConfigFetcherCompletion)(NSData *data, NSURLResponse *response, NSError *error);
 
+/// Completion handler invoked after a fetch that contains the updated keys
+typedef void (^RCNConfigFetchCompletion)(FIRRemoteConfigFetchStatus status,
```

Comment:
thinking: we need a new completion type here since `FIRRemoteConfigFetchCompletion` is used both publicly and internally